backend/app/services/weather_retriever.rb

```diff
@@ -1,6 +1,6 @@
 class WeatherRetriever
   class << self
-    def get(date, postal_code)
+    def get(date, latitude, longitude)
```

Comment:
ncie to not have to go thru postal code for this api

Reply:
doesn't this break the code below? the method expects `postal_code` in a few places

Reply:
Whoops, yeah I meant to remove `postal_code` in those places and replace it with coordinates.